src/main/java/StringManipulation.java

```diff
@@ -1,33 +1,179 @@
 public class StringManipulation implements StringManipulationInterface {
-
+    String myString;
     @Override
     public String getString() {
         return null;
     }
 
     @Override
     public void setString(String string) {
+        this.myString = string;
     }
 
     @Override
     public int count() {
-        return 0;
+        int tally = 0;
+        if(myString == null || myString.length() == 0){
```

Comment:
I feel like this can be shorter for counting.

Reply:
Can you elaborate?